cf.md

```diff
@@ -837,9 +837,14 @@ that need to be preserved:
 
   1. The Cloud Controller Database
   2. The UAA Database
-  3. The Blobstore
-
-The first two are required.  The third is highly recommended, but can be
+  3. The Diego Database
+  4. The CF Networking Database
+  5. The Locket Database
+  6. The Routing Database
+  7. The Autoscaler Database if you have autoscaler enabled with CF deployment
+  8. The Blobstore
+
+The first seven are required.  The third is highly recommended, but can be
```

Comment:
The eighth (Bloblstore) is now the one that is highly recommended. Additionally, since blobstore can be quite large, it might be worth a paragraph with just a bit more guidance around the trade-off between backups with rapid-restore and users re-pushing applications. For large a Blobstore using backup/restore, specialized rapid backup tooling (additional expense) might be necessary.

Reply:
Thanks, good inputs, will update that section accordingly

Reply:
Pushed the commit to improve this section